### ISSUE TITLE
Add ability to take stills

### ISSUE THREAD

**Issue by catdogmat** [schoolpost/CinePI#19]
It would be awesome if you could take still photos with this camera. Please consider adding a button for photos and possibly settings for shutter speed and iso.

**Comment by schoolpost** [comment]
Yes, taking still frames as .DNG file is possible; but is not fully implemented in the cinepi framework yet.

**Comment by mrjohnc** [comment]
Adding this function might help encourage more people to take part in the project, there are a large number of open source people interested in photography and open source cameras
